### PR TITLE
Fix golint for kubectl logs

### PR DESCRIPTION
**Pull request by horis233**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
This PR fixes golint issues in staging/src/k8s.io/kubectl/pkg/cmd/top

1. Add comments for exported functions.
2. Remove `Logs` prefix in `LogsOptions`.

**Which issue(s) this PR fixes**:

Ref #68026

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: staging/src/k8s.io/kubectl/pkg/cmd/logs/logs.go [hunk 1]
```diff
@@ -127,8 +128,9 @@ type LogsOptions struct {
 	containerNameFromRefSpecRegexp *regexp.Regexp
 }
 
-func NewLogsOptions(streams genericclioptions.IOStreams, allContainers bool) *LogsOptions {
-	return &LogsOptions{
+// NewLogsOptions returns a default LogOptions.
+func NewLogsOptions(streams genericclioptions.IOStreams, allContainers bool) *Options {
+	return &Options{
```
Comment (cwdsuzhou) [inline staging/src/k8s.io/kubectl/pkg/cmd/logs/logs.go]: Change to `NewOptions` may be better

Comment (horis233) [reply staging/src/k8s.io/kubectl/pkg/cmd/logs/logs.go]: Thanks! changed `NewLogsOptions` to `NewOptions`

**Review by cwdsuzhou — APPROVED** [review]

**Pull request closed without merging**